src/components/Home/aboutUs.scss

```diff
@@ -5,6 +5,8 @@
     padding: 3rem 0;
 
     .about-container {
+        display: flex;
+        justify-content: space-between;
```

Comment:
I think a `text-align: justify` css is enough to fix this issue. This can be added in the   `.about-content__description` CSS class. What do you say @anubhav1206 ? It is simpler and does the job without adding more code to the project.

Reply:
Could you please provide screenshot?

Reply:
> I think a `text-align: justify` css is enough to fix this issue. This can be added in the `.about-content__description` CSS class. What do you say @anubhav1206 ? It is simpler and does the job without adding more code to the project.

Agreed. text-align: justify works.

Reply:
Are you going to make the change?

Reply:
made the changes.

Reply:
sorry for the delay